src/config/QuoteList.ts

```diff
@@ -553,4 +553,18 @@ export const QuoteList: Quote[] = [
     text: "I am no longer accepting the things I cannot change. I am changing the things I cannot accept.",
     author: "Angela Davis",
   },
-];
+  {
+    text: "Sometimes you can't see what you're learning until you come out the other side.",
+    author: "Wonder Woman",
+  },
+  {
+    text: "Life is not a series of gig lamps symmetrically arranged; \
+	life is a luminous halo, a semitransparent envelope surrounding us \
+	from the beginning of consciousness to the end.",
+    author: "Virginia Woolf",
```

Comment:
```suggestion
    text: "Life is not a series of gig lamps symmetrically arranged;\nlife is a luminous halo, a semitransparent envelope surrounding us\nfrom the beginning of consciousness to the end.",
    author: "Virginia Woolf",
```
The bot is configured to parse `\n` for line breaks 🙂